tshirts/tshirts.cs

```diff
@@ -1,21 +1,38 @@
 ﻿using System;
 using System.Diagnostics;
 
-namespace TshirtSpace {
-    class Tshirt {
-        static string Size(int cms) {
-            if(cms < 38) {
-                return "S";
-            } else if(cms > 38 && cms < 42) {
-                return "M";
-            } else {
-                return "L";
-            }
+namespace TshirtSpace 
+{
+    class Tshirt 
+    {
+        static string Size(int cms) 
+        {           
+                if(cms>0)
+                {
+                    if(cms < 38) 
+                    {
+                        return "S";
+                    } 
+                    else if(cms > 38 && cms < 42) 
+                    {
+                        return "M";
+                    } 
+                    else 
+                    {
+                        return "L";
+                    }
+                }
+                else
+                {
+                    return "Invalid Choice";
+                }            
         }
-        static void Main(string[] args) {
-            Debug.Assert(Size(37) == "S");
-            Debug.Assert(Size(40) == "M");
+        static void Main(string[] args) 
+        {
+            Debug.Assert(Size(38) == "S");

```

Comment:
All boundary values and values between these boundary values need to be verified. ex. 37, 40
In this way you explicitly define the expectation from the production code.